Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -432,17 +434,25 @@ driverAnalyticsView state push =
     , linearLayout  
       [ width MATCH_PARENT
       , height WRAP_CONTENT
-      , margin $ Margin 0 12 0 12
-      , background Color.blue600
+      , margin if getValueFromConfig "BONUS_EARNED" == "true" then (Margin 0 12 0 12) else (Margin 0 4 0 12)
```

Comment:
use `let-in` to get value from function and store in variable

`
let variable = getValueFromConfig "BONUS_EARNED" == "true"
in
  linearLayout[][]
`

Reply:
and use marginVertical